src/layout/Content/Content.js

```diff
@@ -53,13 +56,36 @@ const Content = ({
   const [expand, setExpand] = useState(false)
   const [groupedByName, setGroupedByName] = useState({})
   const [groupedByWorkflow, setGroupedByWorkflow] = useState({})
+  const [showRegisterDialog, setShowRegosterDialog] = useState(false)
+  const location = useLocation()
 
   const contentClassName = classnames(
     'content',
     [JOBS_PAGE, FEATURE_STORE_PAGE, MODELS_PAGE].includes(pageData.page) &&
       'content_with-menu'
   )
 
+  useEffect(() => {
+    if (
+      [
+        PROJECTS_PAGE,
+        ARTIFACTS_PAGE,
+        FILES_PAGE,
+        MODELS_PAGE,
+        FEATURE_STORE_PAGE
+      ].includes(pageData.page) &&
+      ![FEATURES_TAB, MODEL_ENDPOINTS_TAB].includes(match.params.pageTab) &&
+      (![FEATURE_SETS_TAB, FEATURE_VECTORS_TAB].includes(
+        match.params.pageTab
+      ) ||
+        location.search.includes('demo=true'))
```

Comment:
If the query parameters are: `?a=1&b=2&mydemo=true123&c=3` this condition will return `true`, and the button will be displayed, unexpectedly.

How about:
```suggestion
        new URLSearchParams(location.search).get('demo') === 'true'
```
?